port/unix/omrsysinfo.c

```diff
@@ -146,23 +146,25 @@
 #endif
 #endif
 
+#include <unistd.h>
+
 #if defined(LINUX) && !defined(OMRZTPF)
 #include <linux/magic.h>
 #include <sys/sysinfo.h>
 #include <sys/vfs.h>
 #include <sched.h>
-#elif defined(OSX)
+#include "portnls.h"
```

Comment:
This include should be unconditional.

Reply:
Made the include unconditional. It was conditional before because the NLS message was used in function surrounded by `defined(LINUX) && !defined(OMRZTPF)`.